polytope/polytope.py

```diff
@@ -201,19 +201,40 @@ def __copy__(self):
         P.fulldim = self.fulldim
         return P
 
+    def contains(self, point, abs_tol=ABS_TOL):
+        """Return True if polytope contains points.
+
+        @type point: array_like, 1d array, or 2d array (of column vectors)
+
+        @rtype: bool, 1d array
+
+        See Also
+        ========
+        L{is_inside}
+        """
+        if not isinstance(point, np.ndarray):
+            point = np.array(point)
+        if point.ndim == 1:
+            point.shape = (point.size, 1)
+        assert point.shape[0] == self.dim, "points should be col vectors"
+        test = self.A.dot(point) - self.b.reshape((self.b.size, 1)) < abs_tol
+        return np.all(test, axis=0)
+
     def __contains__(self, point, abs_tol=ABS_TOL):
         """Return True if polytope contains point.
 
         See Also
         ========
-        L{is_inside}
+        L{is_inside}, L{Polytope.contains}
         """
         if not isinstance(point, np.ndarray):
             point = np.array(point)
         test = self.A.dot(point.flatten()) - self.b < abs_tol
         return np.all(test)
 
     def are_inside(self, points, abs_tol=ABS_TOL):
+        warnings.warn('polytope.are_inside is deprecated.'
```

Comment:
I can change the docs to use the phrase "strictly contains". How would one express a nonstrict inequality (`<=`)?